eip-1646.md

```diff
@@ -0,0 +1,64 @@
+
+When the third party management is asked with a lag time, the parties that manage the multiple signatures in the aforementioned multiple-signature mechanism have no access control over the user's zero-identity without authorization of the user's backup zero-identity no matter what access rights this backup zero-identity has.
+
+## Rationale
+Although blockchain based decentralized autonomy is still in its early stages it has assumed unprededented achievements. This leads us to a strong belief that decentralized autonomy will not only be blockchain's core value but also be a fundamental force that drives blockchain's development and growth.
+
+To achieve decentralized autonomy, a solution to building an identity based on trustlessness and autonomy is needed. This solution may have no similarities with any existing identity management systems in our society. Bitcoin proposes such a solution by introducing a zero-knowledge proof based mechanism which implements identity anonymity by representing an identity in terms of three elements: a private key, a public key and an address. This mechanism gives full access control over an identity uniquely to its owner. This tells that zero-identity based autonomy can effectively protect a user from privacy disclosure. And only this kind of mechanism can achieve de-privacy in blockchain, then it will make blockchain data publicly accessed without concerns about privacy disclosure therefore make blockchain data a public big data system worldwide.
+
+A user himself/herself is the owner of his/her zero-identity. The user can revoke his/her authorization to an authorized third party. However for a minor or person who needs a guardian since he/she may not be competent to fulfill a responsibility, full access control over his/her zero-identity may be given to a third party e.g. his/her guardian while he/she may only be a user of his/her zero-identity. When he/she becomes an adult or a person that no longer needs a guardian full access control over his/her zero-identity will be transferred  to him/her with smart contracts.  
+
+## Backwards Compatibility
+There is no backwards compatibility issue. 
+
+## Implementation
+The reference implementation for ERC-1484 may be found in [Silkroad-Framework/eip](https://github.com/Silkroad-Framework/eip).
```

Comment:
This still references ERC-1484, which seems to be a copy/paste from https://eips.ethereum.org/EIPS/eip-1484